complex_functions/deutsch_jozsa.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
Can you add the small comment saying that this function is not ours and belongs to the Python repo. See my files for the comment line

Reply:
Great suggestion, I'll add that.